.github/workflows/docker.yml

```diff
@@ -0,0 +1,40 @@
+name: Build-Env Docker Image
+
+on:
+  push:
+    branches: master
+    paths:
+    - '**Dockerfile'
+    - '**docker.yml'
+#  workflow_dispatch:
+#  schedule:
+#    - cron: '30 11 20 */3 *'
```

Comment:
How often would you like this automatically built?

Reply:
Do you have any suggestion for that?
I think once each fortnight is probably enough, possibly even too much. 
I mean, if there are good reasons to have a new image asap we can just trigger it manually.

Reply:
At first I thought a fortnight as well, although since I'm an American I actually thought two weeks. ;) But... monthly should be alright, because as you said, we can trigger it manually, or change the cron setting